packages/cookie-banner/__tests__/form/render.js

```diff
@@ -51,4 +51,55 @@ describe(`Cookie banner > DOM > form > render`, () => {
+        <p>Cookies are a common feature used on almost all websites. A cookie is a small text file that is downloaded onto a device to allow a website to recognise it and to store some information about your preferences.</p>
+        <p>We use cookies for different reasons, including for managing your shopping cart, measuring your visits on our sites, recognising and remembering your preferences and showing you personalised ads. Learn more about cookies and how we use them.</p>
+        <p>Cookies are a common feature used on almost all websites. A cookie is a small text file that is downloaded onto a device to allow a website to recognise it and to store some information about your preferences.</p>
+        <p>We use cookies for different reasons, including for managing your shopping cart, measuring your visits on our sites, recognising and remembering your preferences and showing you personalised ads. Learn more about cookies and how we use them.</p>
+        <p>Cookies are a common feature used on almost all websites. A cookie is a small text file that is downloaded onto a device to allow a website to recognise it and to store some information about your preferences.</p>
+        <p>We use cookies for different reasons, including for managing your shopping cart, measuring your visits on our sites, recognising and remembering your preferences and showing you personalised ads. Learn more about cookies and how we use them.</p><div class="privacy-banner__form-container"></div>`;
+        cookieBanner({
+            secure: false,
+            hideBannerOnFormPage: false,
+            types: {
+                test: {
+                    suggested: true,
+                    title: 'Test title',
+                    description: 'Test description',
+                    labels: {
+                        yes: 'Test yes label',
+                        no: 'Test no label'
+                    },
+                    fns: [
+                        () => { }
+                    ]
+                },
+                performance: {
+                    title: 'Performance preferences',
+                    description: 'Performance cookies are used to measure the performance of our website and make improvements.',
+                    labels: {
+                        yes: 'Pages you visit and actions you take will be measured and used to improve the service',
+                        no: 'Pages you visit and actions you take will not be measured and used to improve the service'
+                    },
+                    fns: [
+                        () => { }
+                    ]
+                }
+            }
+        });
+        const top = document.querySelector(`.${defaults.classNames.form}`).offsetTop
+        
+        expect(window.scrollY).toEqual(top);
+    });
+

```

Comment:
I don't think that JSDOM supports layout calculations - https://github.com/jsdom/jsdom/issues/135 - so we may not be able to write a unit test to cover this.

Reply:
Cool - yes, I wasn't sure if this would do anything.  I've removed the test.